content/ops_guides/backup_restore_guide.md

```diff
@@ -102,11 +102,11 @@ Snapshots of the Cloudify HA cluster should be taken at regular intervals (sugge
     "http://<manager-ip>/api/v3.1/snapshots/<snapshot-id>/restore"
     ```
 
-    Parameters specification available in the [Cloudify API documentation](http://docs.cloudify.co/api/latest/#restore-snapshot).
+    Parameters specification available in the [{{< param product_name >}} API documentation](http://docs.cloudify.co/api/latest/#restore-snapshot).
 
 1. Snapshot-restore status
 
-    **(Supported for Cloudify Manager 5.0.5 and above.)**
+    **(Supported for {{< param cfy_manager_name >}} 5.0.5 and above.)**
```

Comment:
In other places you removed references to specific versions. Maybe here it should be removed as well?

Reply:
agree. fixed.